data/sles4sap/sap_deployment_automation_framework/SAP_SYSTEM.tfvars

```diff
@@ -868,4 +868,5 @@ bom_name = "%SDAF_BOM_NAME%"
 # These tags will be applied to all resources
 tags = {
   "DeployedBy" = "OpenQA-SDAF-automation",
+  %SDAF_NO_CLEANUP%
```

Comment:
Temporary solution. 
Proper one in progress: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21611